src/compiler/crystal/command.cr

```diff
@@ -64,7 +66,7 @@ class Crystal::Command
     when !command
       puts USAGE
       exit
-    when "init".starts_with?(command)
+    when command == "init"
```

Comment:
This is technically a breaking change: if you did `crystal i` before it was the same as `crystal init`. Maybe it's fine, though? I don't know how many knew this.

Reply:
I think is fine, it been like that since the beginning but I never saw it used.